packages/coil-extension/src/popup/mocks/loadMockedStates.tsx

```diff
@@ -226,7 +226,7 @@ const aliceUnsubscribed = mockState({
 
 const MOCK_STATES = [
   { name: 'New Extension UI', state: userCanTipNewUi },
-  { name: 'New Extension UI Compare', state: userCanTipNewUiCompare },
+  // { name: 'New Extension UI Compare', state: userCanTipNewUiCompare },
```

Comment:
Do we want to leave these comments for now or just remove?

Reply:
The load mocked file is almost irrelevant. It's only for spinning up a dev view so it changes all the time depending on how you want it to look.

Reply:
Yeah, it's kinda broken window theory in action, 
but please remove or it's only going to get worse

Reply:
> Built and ran and the animations work fine. It did seem to get in a weird wonky state once, where there was no tab visible (just empty whitespace) but I couldn't reproduce it.

Thanks, ill try throttling my network and see if that helps reproduce it. By "no tab visible" do you mean the main navigation ( 3 bottom tabs ) was not visible?